main.cpp

```diff
@@ -124,12 +128,13 @@ static const AVal av_NetStream_Play_Start = AVC("NetStream.Play.Start");
 static const AVal av_Started_playing = AVC("Started playing");
 static const AVal av_NetStream_Play_Stop = AVC("NetStream.Play.Stop");
 static const AVal av_Stopped_playing = AVC("Stopped playing");
-
 static const AVal av_OBSVersion = AVC("TwitchTest/1.3");
 static const AVal av_setDataFrame = AVC("@setDataFrame");
+static json_t *ingests;
+static int *indexOrder = NULL;
 
```

Comment:
Global variables for sharing between the bandwidth test and populate list functions.